CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Added
 
+- Support for fallback locale when using `RRule::humanReadable()` [#11](https://github.com/rlanvin/php-rrule/pull/11)
```

Comment:
Please help me out with the `CHANGELOG.md`, is this the correct place to put this string? Should a new minor version title be created? I guess you'd handle that.